src/binfhe/examples/boolean-serial-binary-dynamic-large-precision.cpp

```diff
@@ -0,0 +1,213 @@
+//==================================================================================
```

Comment:
The files boolean-serial-binary-dynamic-large-precision.cpp and boolean-serial-json-dynamic-large-precision.cpp are almost identical. The only difference here is in the serialization: BINARY or JSON. you should create one function and call it with different SerType.

Reply:
I'm actually not sure how to address this. In [this file](https://github.com/openfheorg/openfhe-development/blob/main/src/core/include/utils/sertype.h), BINARY and JSON doesn't seem to be in the same type or done by enum. Thanks!

Reply:
This is an example. It is better to keep examples simpler so that users could copy them, I don't think this change is necessary.